index.html

```diff
@@ -242,12 +244,37 @@ <h2 class="language">Skills</h2>
 
 
             </section>
-            
+
+
         </section>
     </div>
+    <section class="white">
+    <footer class="footer white">
+        <div class="foot">
+        <h2 class="contact-text">Contact me</h2>
+        <p class="contact-paragraph">
+            If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it
+        </p>
+        <form>
+            <label for="fname">Full name:</label>
+            <input id="fname" type="text" maxlength="30" name="full-name" placeholder="Yeremias" required class="input">
+
+            <label for="email">Email</label>
+            <input id="email" type="email" name="email-address" placeholder="natajayanj14@gmail.com" required class="input">
+
+            <label for="textarea">Message:</label>
+            <textarea id="textarea" name="comment" maxlength="500" cols="30" rows="10" placeholder="Write your message here" required class="input text-area"></textarea>
```

Comment:
- [ ] The `input` field  should have a `length` and should also be `required` see screenshot 👇
![zzz](https://user-images.githubusercontent.com/61562626/129794413-27bdfb2e-7078-41c1-99e2-f67239996735.jpeg)

Reply:
- The input field does have a max length of 30 and the message input has a max length of 500. 

- Every field in the form is also required

- Email input type is email